posthog/migrations/0282_prompt_sequence_model.py

```diff
@@ -0,0 +1,78 @@
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0281_create_insight_caching_state_model"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="Prompt",
+            fields=[
+                ("id", models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                ("step", models.IntegerField()),
+                ("type", models.CharField(max_length=200)),
+                ("title", models.CharField(max_length=200)),
+                ("text", models.CharField(max_length=1000)),
+                ("placement", models.CharField(default="top", max_length=200)),
+                ("buttons", models.JSONField()),
+                ("reference", models.CharField(default=None, max_length=200, null=True)),
+                ("icon", models.CharField(max_length=200)),
+            ],
+        ),
+        migrations.CreateModel(
+            name="PromptSequence",
+            fields=[
+                ("id", models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                ("key", models.CharField(max_length=200)),
+                ("type", models.CharField(max_length=200)),
+                (
+                    "path_match",
+                    django.contrib.postgres.fields.ArrayField(base_field=models.CharField(max_length=200), size=None),
+                ),
+                (
+                    "path_exclude",
+                    django.contrib.postgres.fields.ArrayField(base_field=models.CharField(max_length=200), size=None),
+                ),
+                ("status", models.CharField(max_length=200)),
+                ("requires_opt_in", models.BooleanField(default=False)),
+                ("autorun", models.BooleanField(default=True)),
+                ("must_have_completed", models.ManyToManyField(blank=True, to="posthog.PromptSequence")),
+                ("prompts", models.ManyToManyField(to="posthog.Prompt")),
+            ],
+        ),
+        migrations.CreateModel(
+            name="UserPromptState",
+            fields=[
+                ("id", models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                ("last_updated_at", models.DateTimeField(default=django.utils.timezone.now)),
+                ("step", models.IntegerField(default=None, null=True)),
+                ("completed", models.BooleanField(default=False)),
+                ("dismissed", models.BooleanField(default=False)),
+                (
+                    "sequence",
+                    models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to="posthog.promptsequence"),
+                ),
+                ("user", models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
+            ],
+        ),
+        migrations.DeleteModel(
+            name="UserPromptSequenceState",
+        ),
```

Comment:
What's the migration path considering `UserPromptSequenceState` is deleted?

Reply:
There's no need for one, the old model has being used for prompts that are deprecated, so we can just delete and start fresh